src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Native lazy loading for the web
+subhead: Browser-level native lazy loading is finally here!
+authors:
+  - houssein
+  - addyosmani
+  - mathiasbynens
+date: 2019-07-17
+hero: hero.png
+alt: Phone outline with loading image and assets
+description: |
+  This post covers the loading attribute and how it can be used
+  to control the loading of images and iframes.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+Support for natively lazy loading images and iframes is coming to the web! This video shows
+a [demo](https://mathiasbynens.be/demo/img-loading-lazy) of the feature:
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="./lazyload.webm" type="video/webm">
+    <source src="./lazyload.mp4" type="video/mp4">
+  </video>
+</figure>
+
+Starting with Chrome 76, you'll be able to use the new `loading` attribute to lazy load resources
```

Comment:
Should it be "to lazily load?"

Reply:
Good catch. I think there's a larger revision to be made here, actually. I had proposed removing the hyphen in "lazy-load," but @addyosmani said he'd like to standardize on the hyphenated version. So, I need to go through and fix that. I'll add a commit in just a bit. (Sorry for the unnecessary trouble, @housseindjirdeh!)

Reply:
Good catch. I had proposed going with "lazy load," but @addyosmani suggested we standardize on "lazy-load" (with a hyphen), so I've added a commit to take care of that throughout the post. Sorry for the extra trouble, @housseindjirdeh!

Reply:
No worries at all @mfriesenhahn, and thanks for updating it again throughout the article 👍